src/components/date-picker/pickers/Picker.ts

```diff
@@ -115,6 +117,24 @@ export abstract class Picker {
         return moment(date).isoWeek();
     }
 
+    private get formatDate() {
+        const longDateFormat = new Intl.DateTimeFormat(this.language, {
+            dateStyle: 'long',
+        });
+
+        return (date: Date | null, format: string): string => {
+            if (!date) {
+                return '';
+            }
+
+            if (format === ARIA_DATE_FORMAT) {
+                return longDateFormat.format(date);
+            }
```

Comment:
I think think might need an explanatory comment in the code… 😅

Reply:
Ah… if I understand the code correctly, flatpickr is supplying the `format` argument, and when it tries to format a value in order to use it for an ARIA attribute, the `format` argument's value will be `'F j, Y'`. Correct?

Do you know if the "long" format we return is equivalent to `'F j, Y'`?

Reply:
I felt like this was a relatively small thing, so I merged the fix anyway. But if you feel like you can write a helpful comment, or refactor with a named function in a useful way, please feel encouraged to do so! 😊

Reply:
> Ah… if I understand the code correctly, flatpickr is supplying the `format` argument, and when it tries to format a value in order to use it for an ARIA attribute, the `format` argument's value will be `'F j, Y'`. Correct?
> 
> Do you know if the "long" format we return is equivalent to `'F j, Y'`?

Correct. Yep, the long date style corresponds to "January 1, 2000" but localized. So the idea is that this will be more fit as an accessible label than a numeric date, and including time is also not meaningful here.

From what I could see those labels are only used when picking the dates of a month when the type of pickers is either "date" or "datetime".